src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -117,6 +121,14 @@ def create_webapp(cmd, resource_group_name, name, plan, runtime=None, startup_fi
     return webapp
 
 
+def validate_linux_create_options(runtime=None, deployment_container_image_name=None,
+                                  multicontainer_config_type=None, multicontainer_config_file=None):
+    if bool(multicontainer_config_type) != bool(multicontainer_config_file):
+        return False
+    opts = [bool(runtime), bool(deployment_container_image_name), bool(multicontainer_config_type)]
```

Comment:
Minor: you can remove the `bool()` to leverage Python's support for [True Value testing](https://docs.python.org/2.4/lib/truth.html)